data/yam/autoyast/create_hdd_gnome_x86_64_raid_encrypted_home.xml

```diff
@@ -0,0 +1,351 @@
+      <disklabel>gpt</disklabel>
+      <partitions t="list">
+        <partition t="map">
+          <create t="boolean">true</create>
+          <format t="boolean">false</format>
+          <partition_id t="integer">263</partition_id>
+          <partition_nr t="integer">1</partition_nr>
+          <resize t="boolean">false</resize>
+          <size>8388608</size>
+        </partition>
+        <partition t="map">
+          <create t="boolean">true</create>
+          <format t="boolean">false</format>
+          <partition_id t="integer">253</partition_id>
+          <partition_nr t="integer">2</partition_nr>
+          <raid_name>/dev/md0</raid_name>
+          <resize t="boolean">false</resize>
+          <size>13524533248</size>
+        </partition>
+        <partition t="map">
+          <create t="boolean">true</create>
+          <format t="boolean">false</format>
+          <partition_id t="integer">253</partition_id>
+          <partition_nr t="integer">3</partition_nr>
+          <raid_name>/dev/md1</raid_name>
+          <resize t="boolean">false</resize>
+          <size>5793382400</size>
+        </partition>
+        <partition t="map">
+          <create t="boolean">true</create>
+          <format t="boolean">false</format>
+          <partition_id t="integer">253</partition_id>
+          <partition_nr t="integer">4</partition_nr>
+          <raid_name>/dev/md2</raid_name>
+          <resize t="boolean">false</resize>
+          <size>2147466752</size>
+        </partition>
+      </partitions>
+      <type t="symbol">CT_DISK</type>
+      <use>all</use>
+    </drive>
+    <drive t="map">
+      <device>/dev/vdb</device>
+      <disklabel>gpt</disklabel>
+      <partitions t="list">
+        <partition t="map">
+          <create t="boolean">true</create>
+          <format t="boolean">false</format>
+          <partition_id t="integer">253</partition_id>
+          <partition_nr t="integer">1</partition_nr>
+          <raid_name>/dev/md0</raid_name>
+          <resize t="boolean">false</resize>
+          <size>13524533248</size>
+        </partition>
+        <partition t="map">
+          <create t="boolean">true</create>
+          <format t="boolean">false</format>
+          <partition_id t="integer">253</partition_id>
+          <partition_nr t="integer">2</partition_nr>
+          <raid_name>/dev/md1</raid_name>
+          <resize t="boolean">false</resize>
+          <size>5793382400</size>
+        </partition>
+        <partition t="map">
+          <create t="boolean">true</create>
+          <format t="boolean">false</format>
+          <partition_id t="integer">253</partition_id>
+          <partition_nr t="integer">3</partition_nr>
+          <raid_name>/dev/md2</raid_name>
+          <resize t="boolean">false</resize>
+          <size>2147466752</size>
+        </partition>
+      </partitions>
+      <type t="symbol">CT_DISK</type>
+      <use>all</use>
+    </drive>
+  </partitioning>
+  <proxy t="map">
```

Comment:
probably proxy is also default, and can be drop, compare with autoyast_mini.

Reply:
removed.